Autopilot/PathManager/Inc/landingTakeoffManager.hpp

```diff
@@ -0,0 +1,170 @@
+#pragma once
+
+/* Related Confluence Pages
+    https://uwarg-docs.atlassian.net/wiki/spaces/ZP/pages/1607630921/Landing+and+Takeoff
+*/
+
+#include "waypointManager.hpp"
+#include "vectorClass.hpp"
```

Comment:
vector class does not seem to get used in this header file, right ? If that's the case then don't include it in the header in an effort to reduce dependancies. Include it in the cpp instead.

Is the same true with AutoSteer ?

Reply:
not for autosteer, the coordturns translation command needs the struct